### ISSUE TITLE
How to provide multiple confidence interval in prophet?

### ISSUE THREAD

**Issue by nikitagupta55** [facebookincubator/prophet#266]
As, of now it's something like this : m <- prophet(df, interval.width = 0.95)

Can I give multiple confidence interval values like : m <- prophet(df, interval.width = c(0.75,.95))  ?

**Comment by bletham** [comment]
No, but you can get finer grained information about the posterior using the `predictive_samples` function from the v0.2 branch (see documentation here: https://github.com/facebookincubator/prophet/blob/v0.2/R/R/prophet.R#L939). You pass this the same inputs as `predict`, but it will return all of the posterior samples rather than quantiles. You can then compute quantiles as you wish.

You can install the v0.2 branch from github using devtools:
```
devtools::install_github('facebookincubator/prophet', subdir='R', ref='v0.2')
```

**Comment by bletham** [comment]
This is now available in v0.2 in CRAN.
